src/components/Header/Header.css

```diff
@@ -1,7 +1,32 @@
 .header {
     position: fixed;
     width: 100vw;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    margin-top: 30px;
 }
 
-/* TODO: Add styling for your Header component
-here. */
\ No newline at end of file
+#my-profile {
+    margin-right: 50px;
+    font-size: 24px;
```

Comment:
for scalability, it's best to use units like `em` or even % over `px` unless you want something to remain a specific size no matter what, as noted [here](https://elementor.com/help/whats-the-difference-between-px-em-rem-vw-and-vh/) (specifically the last section). let's change any `px` to make sure the page still follows the Figma even if it's on different-sized screens or zoomed in/out.

Reply:
nitpick: the `My Profile` text font size should be smaller than the Toody logo.